web-app/packages/lib/src/modules/project/components/ProjectMembersTable.vue

```diff
@@ -186,6 +196,10 @@ const searchedItems = computed(() =>
   })
 )
 
+function canRemoveMember(item: ProjectAccessDetail) {
```

Comment:
How is 'leave a project' handled now? Can you leave project anytime regardless of other members, e.g. you are the last member or project.creator.

Reply:
If you are guest, you can leave project.

```ProjectViewTemplate.vue#251```

Reply:
In CE if everyone is a guest (no global permission is set) I can leave project anytime, right? Even if I created it.